openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -741,3 +741,22 @@ def test_course_enrollment_status(self, start_offset, end_offset, is_enrollment_
         data = utils.supplement_program_data(self.program, self.user)
 
         self._assert_supplemented(data, is_enrollment_open=is_enrollment_open)
+
+    @mock.patch(UTILS_MODULE + '.get_organization_by_short_name')
+    def test_organizations_logo_exists(self, mock_get_organization_by_short_name):
+        """ Verify the logo image is set from the organizations api """
+        mock_logo_url = 'edx/logo.png'
+        mock_image = mock.Mock()
+        mock_image.url = mock_logo_url
+        mock_get_organization_by_short_name.return_value = {
+            'logo': mock_image
```

Comment:
Please also include a test covering the case where the organization is present, but the logo is missing.